### PR TITLE
added: textutilities

### PR DESCRIPTION
**Pull request by sharmayatendra**
# Added: TextUtilities

## Textutilities of different types added.

image
image
image

### CODE REVIEW

**Review by vikrantkode — COMMENTED** [review]
-Code is cleanly written
-You can merge it

File: components/TextUtilities/textutilities.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    --grey-text-color: rgb(128, 128, 128);
+}
+
+.textutil-container {
+    border: 1px solid var(--primary-color);
+    padding: 1rem;
+}
+
+.heading , .small-text , .centered-text{
+    margin: 0.5rem 0 0.5rem 0;
```
Comment (yashpatel1797) [inline components/TextUtilities/textutilities.css]: also do this margin: 0.5rem 0; it will be the same.

File: css/main.css [hunk 2]
```diff
@@ -35,11 +35,21 @@ a {
...
     list-style: none;
     font-size: 1.2em;
-    margin: 6px 2px;
-    padding: 4px;
+    margin: 4px 2px;
+    padding: 2px;
+}
+
+.nav-list > li:hover {
+    background-color: rgb(228, 228, 228);
```
Comment (yashpatel1797) [inline css/main.css]: use variable here also.

**Review by yashpatel1797 — APPROVED** [review]
added some suggestions everything else looks good.

File: components/TextUtilities/textutilities.css (hunk 1 above)
Comment (sharmayatendra) [reply components/TextUtilities/textutilities.css]: done

File: css/main.css (hunk 2 above)
Comment (sharmayatendra) [reply css/main.css]: done

**Pull request merged**